pkg/server/container_execsync.go

```diff
@@ -55,6 +56,22 @@ func (c *criContainerdService) ExecSync(ctx context.Context, r *runtime.ExecSync
 		return nil, fmt.Errorf("container %q is in %s state", id, criContainerStateToString(meta.State()))
 	}
 
+	// Get exec process spec.
```

Comment:
see the comment above for call to get metadata for the container from the container metadata store